scripts/prompter.js

```diff
@@ -0,0 +1,108 @@
+// OPD-1A
+// const salesCount = departments.sales.numberOfEmployees;
+// console.log("De afdeling Sales heeft " + [salesCount] + " medewerkers")
+//
+// OPD-1B
+// const marketingDescription = departments.marketing.description;
+// console.log("Marketing is een leuke afdeling om te werken. " + [marketingDescription] + ".")
+//
+// OPD-1C
+// const csCount = departments["customer-service"].numberOfEmployees;
+// console.log("De afdeling Customer Service heeft " + [csCount] + " medewerkers")
+//
+// OPD-1D
+// const funcDescSales = departments.sales.jobs[1].description;
+// console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + [funcDescSales]);
+//
+// OPD-2A-B-C
+const departmentInfo = prompt(`Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]`);
+switch (departmentInfo) {
+    case "marketing":
+        // console.log("Je koos " + [departmentInfo] + ". " + departments.marketing.description);
+        break;
+    case "sales":
+        // console.log("Je koos " + [departmentInfo] + ". " + departments.sales.description);
+        break;
+    case "customer-service":
+        // console.log("Je koos " + [departmentInfo] + ". " + departments["customer-service"].description);
+        break;
```

Comment:
Dit had je nog iets kunnen versimpelen
```suggestion
    case "marketing":
    case "sales":
    case "customer-service":
        // console.log("Je koos " + [departmentInfo] + ". " + departments[departmentInfo].description);
        break;
```

Reply:
@Zwen1111 Thanks voor de feedback. Is een stuk schoner zo. Aangepast. Pull request vervolgens maar gemerged